SalesforceSDK/Salesforce.SDK.Phone/Source/Pages/AccountPage.xaml.cs

```diff
@@ -115,7 +119,11 @@ public void ContinueWebAuthentication(WebAuthenticationBrokerContinuationEventAr
         private void DisplayErrorDialog(string message)
         {
             MessageContent.Text = message;
```

Comment:
Technically setting the text is a UI action, it should be moved over to the Dispatcher. It could also be the cause for the ArgumentException.